watchdog_linux.go

```diff
@@ -22,14 +22,14 @@ func init() {
 	// returns the configured WatchdogSec in the service unit as time.Duration
 	interval, err := daemon.SdWatchdogEnabled(false)
 	if err != nil || interval == 0 {
-		log.Printf("Systemd watchdog not enabled")
+		log.Debug("Systemd watchdog not enabled")
 		return
 	}
 
 	// We want to notify the watchdog every WatchdogSec/3, that is, if WatchdogSec is
 	// set to 30 seconds, we'll send a notification to systemd every 10 seconds.
 	runEvery := interval / 3
-	log.Printf("Systemd watchdog notifications every %.2f seconds", runEvery.Seconds())
+	log.Debug("Systemd watchdog notifications every %.2f seconds", runEvery.Seconds())
```

Comment:
We probably need to use a `Debugf` here, as this contains a formatting directive.